specs/unified_kernel_image.md

```diff
@@ -53,14 +53,15 @@ UKIs consist of the following resources:
 * Optionally, a microcode initrd in the `.ucode` section, to be handed to the kernel before any other initrd.
 * Optionally, a splash image to bring to screen before transitioning into the Linux kernel, in the `.splash` section.
 * Optionally, one or more compiled Device Trees, for systems which need it, each in its separate `.dtb` section. If multiple `.dtb` sections exist then one of them is selected according to an implementation-specific algorithm.
+* Optionally, one or more compiled Device Trees, for systems which need it, each in its separate `.dtbauto` section. If multiple `.dtbauto` sections exist then one of them is selected according to an implementation-specific algorithm.
```

Comment:
I think we should define the algorithm here at least in vague terms, i.e. that the devicetree contents is checked and compared with local system devicetree or smbios or equivalent data and matched up.